src/main/java/de/tum/in/www1/artemis/service/compass/grade/CompassGrade.java

```diff
@@ -3,6 +3,7 @@
 import java.util.HashMap;
 import java.util.Map;
 
+// TODO CZ: remove? if parser is removed, CompassResult has to be adjusted and can be used instead of CompassGrade in the CompassService
```

Comment:
Can we remove it?

Reply:
I am not really happy about this class as it is only used once in the `CompassService` after rounding the points and it is really similar to the `CompassResult`. However, I would not replace the usage simply with the `CompassResult` there, as it would result in a `CompassResult` with an invalid state. I remove the TODO for now and think of something in the future.